src/inverseRobot.js

```diff
@@ -7,7 +7,20 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const robotInverse = {};
+  let name;
+
+  for (const key in robot) {
+    name = robot[key];
```

Comment:
I think it would be better if you implement the variable here, and use `const` instead of `let`
```suggestion
    const name = robot[key];
```